src/commands/path/__test__/generate.path.test.ts

```diff
@@ -111,8 +111,8 @@ describe('GeneratePathHistoricImagery', () => {
       epsg: 2193,
     };
     assert.throws(() => {
```

Comment:
Having just `Error` as the throws checker isn't really adding anything, could it be improved with matching the error message?

otherwise you could drop the Error `assert.throws(() => generatePath(metadata))` would be a similar enough check.

Reply:
> Having just `Error` as the throws checker isn't really adding anything, could it be improved with matching the error message?

Absolutely, but I don't feel like that's relevant for this PR.

> otherwise you could drop the Error `assert.throws(() => generatePath(metadata))` would be a similar enough check.

:shrug: I guess that weakens the test a little bit, since it won't check the error type anymore. Is this worth doing?